projects/js-toolkit/packages/liferay-cli/src/new/facet-shared-bundle/templates/src/index.js.ejs

```diff
@@ -0,0 +1,10 @@
+/**
+ * Place any initialization of this shared bundle in this file. That way, you
+ * can run:
+ *
+ * 		require('<%= name %>');
```

Comment:
it may just be github, but these tabs look gigantic

Reply:
:thinking: They are real tab characters (not spaces) so I guess it depends on what's displaying them...

Reply:
I'll review the .ejs templates in any case, to make sure we use tabs/spaces everywhere.

Reply:
Done. We use tabs in .ejs templates (like this one). Sadly prettier do not format these files, nor eslint checks them. They are so few that I don't think they deserve a custom SF process, but if they grow maybe we need to think something.